src/main/java/pl/simpleascoding/tutoringplatform/exception/UserNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package pl.simpleascoding.tutoringplatform.exception;
+
+public class UserNotFoundException extends RuntimeException {
+    private static final String ERROR_WITH_ID = "User with id %d not found";
+    private static final String ERROR_WITH_USERNAME = "User %s not found";
```

Comment:
Please change name of the field to:
-  **MESSAGE_USER_FOR_THIS_NAME_NOT_FOUND**

Reply:
Done